prelude/cxx/cxx_library.bzl

```diff
@@ -1379,8 +1379,7 @@ def _strip_objects(ctx: AnalysisContext, objects: list[Artifact]) -> list[Artifa
     cxx_toolchain_info = get_cxx_toolchain_info(ctx)
 
     # Stripping is not supported on Windows
-    linker_type = cxx_toolchain_info.linker_info.type
-    if linker_type == LinkerType("windows"):
+    if host_info().os.is_windows:
```

Comment:
In general, using `host_info()` is an anti-pattern, because it breaks cross-platform builds. I don't think we want to introduce any usages of `host_info` in these files.